tagger.py.in

```diff
@@ -2,18 +2,20 @@
 
 import os
 import sys
+import multiprocessing
 
 
-sys.path.insert(0, '.')
 
+sys.path.insert(0, '.')
 # This is needed to find resources when using pyinstaller
 if getattr(sys, 'frozen', False):
     basedir = getattr(sys, '_MEIPASS', '')
 else:
     basedir = os.path.dirname(os.path.abspath(__file__))
-
 if sys.platform == 'win32':
```

Comment:
This is a minor thing, but can we keep the empty line before this? Otherwise this looks too much like it belongs to the if above.

Reply:
Sure, my bad.